lib/pages/detail/no_score_message.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:pola_flutter/theme/assets.gen.dart';
+import 'package:pola_flutter/theme/colors.dart';
+import 'package:pola_flutter/theme/fonts.gen.dart';
+import 'package:pola_flutter/theme/text_size.dart';
+import 'package:pola_flutter/i18n/strings.g.dart';
+
+class NoScoreMessage extends StatelessWidget {
+  const NoScoreMessage({Key? key, required this.t}) : super(key: key);
+
+  final Translations t;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 22.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Center(
+            child: Assets.company.unpublished.svg(height: 109.42, width: 109.55),
+          ),
+          const SizedBox(height: 26.0),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 115.0),
+          ),
+          Text(
+            t.companyScreen.companyUnverified,
+            style: TextStyle(
+              fontSize: TextSize.description,
+              fontWeight: FontWeight.w400,
+              fontFamily: FontFamily.lato,
+              color: AppColors.text,
+            ),
+            textAlign: TextAlign.left,
+          ),
+          Text(
+            t.companyScreen.thankYou,
```

Comment:
Skopiowałeś zmiany z innego brancha. I to jest spoko i poprawne. Zapomniałeś tylko, że zmiany były też w pliku z tłumaczeniami. Musisz je uzupełnić i odpalić slanga, żeby wygenerował kod. To samo z `t.companyScreen.companyUnverified`